### ISSUE TITLE
Occasional lack of color

### ISSUE THREAD

**Issue by byorgey** [jtdaugherty/brick#409]
See https://github.com/swarm-game/swarm/issues/815 for context.  Occasionally (maybe 1 time out of 100?) when starting the `swarm` application, it will be displayed without color.  Simply exiting and restarting always resolves the issue.  Multiple developers have experienced the same occasional issue; so far we have not found anything that the occurrences have in common, nor have we found any way to reproduce it.

This seems like a truly strange bug, and I would not blame you for deciding it wasn't worth tracking down (if in fact it is a `brick` or `vty` issue).  But I thought it was worth posting an issue here just in case someone else has seen this behavior before, or has any ideas at all about what might cause it.

**Comment by jtdaugherty** [comment]
I believe this is a duplicate of https://github.com/jtdaugherty/brick/issues/378 (and I'm hoping it is). I have not been able to test my theory on that one, but I did report it to the upstream maintainer. Since I haven't gotten any response there, it might be time to hack on `terminfo` and stop using the unsafe IO it's doing. Of course, it'll be hard to tell if we've resolved this if it is hard to reproduce...

Thanks for reporting this!
